src/components/common/ProfilePostgrid/ProfilePostGrid.module.css

```
@@ -0,0 +1,23 @@
+.grid {
+  padding: 16px;
+  margin: 0 -4px;
+  background-color: var(--color-white);
+  border-radius: 4px;
+  border: 1px solid var(--color-silver);
```

We don't have CSS variables "--color-white" and "--color-silver" in the project

```suggestion
  background-color: white;
  border-radius: 4px;
  border: 1px solid gray;
```